spec/services/proofing/lexis_nexis/ddp/response_redacter_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+describe Proofing::LexisNexis::Ddp::ResponseRedacter do
+
+  let(:json) { 
+    Proofing::LexisNexis::Ddp::ResponseRedacter.
+      redact(sample_hash)
+  }
+
+  describe 'self.redact' do
+    context 'hash with unknown keys' do
+      let(:sample_hash) {
+        {
+          'unknown_key' => 'dangerous data',
+          'first_name' => 'unsafe first name',
+          'ssn_hash' => 'unsafe ssn hash',
+        }
+      }
+      it 'redacts unknown keys' do
+        expect(json.values).to eq(['[redacted]']*3)
+      end
+      it 'keeps redacted keys' do
+        expect(json.keys.length).to eq(3)
+      end
```

Comment:
I'd personally just assert the whole body:
```suggestion
      it 'redacts values for unknown keys' do
        expect(json).to eq(
          'unknown_key' => '[redacted'],
          'first_name' => '[redacted]',
          'ssh_hash' => '[redacted'],
        )
```